v1.21.1 Changelog (#2992)
mattermost/mattermost-mobile: CHANGELOG.md

# Mattermost Mobile Apps Changelog

## 1.21.1 Release
- Release Date: TBD
- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device

### Combatibility
 - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
 - iPhone 5s devices and later with iOS 11+ is required.

### Bug Fixes

>  - Fixing an issue on Android where logging in using GitLab or OneLogin fails to display Mattermost on a subpath server.

Reviewer: ```suggestion
 - Fixing an issue on Android where logging in using SSO fails when the Mattermost server is running on a subpath.
```
